docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -50,6 +50,7 @@ The following configuration properties are available:
 Property name                              Description
 ========================================== ==============================================================
 ``mongodb.connection-url``                 The connection url that the driver uses to connect to a MongoDB deployment
+``mongodb.schema-database``                The database to use for schema management
```

Comment:
Is this config property still needed in your organization?

Reply:
Described the use case in the comment at the end of this thread.